projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -438,6 +461,34 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
           this.chartContainerElement,
           LegendPosition.None
         );
+
+        /*
+        Only uncomment this when wanting to add support for additional series's legend, that too flag controlled.
+        Also, adding this might change the bounds, need to restructure UI and components to eject legend out of the graph to avoid affecting scales.
```

Comment:
Not sure I understand the context or motivation to add this commented code here. Is there a Slack thread or Jira issue for reference with details?